core/sail/federation/src/test/java/org/eclipse/rdf4j/sail/federation/optimizers/CardinalityTest.java

```diff
@@ -0,0 +1,66 @@
+/*******************************************************************************
+ * Copyright (c) ${year} Eclipse RDF4J contributors.
```

Comment:
Looks as if your editor forgot to replace `{$year}` with the actual year!

Reply:
I saw other files, they have "2015" in place of `${year}`. Should I do the same or should I put "2020"?

Reply:
I replaced it with `2020` in the latest commit.